locales/jp.yml

```diff
@@ -1,11 +1,11 @@
 ---
 jp:
-  navigation:
+  global_navigation:
     support: サポート
-    contribute: Contribute
+    community: Community
     docs: Docs
     logo: Middleman
-    search_placeholder: ドキュメント検索
```

Comment:
It appears that it should be used here, but isn’t: https://github.com/middleman/middleman-guides/blob/master/source/partials/_docsearch.erb#L3